addons/project/models/project.py

```diff
@@ -1502,7 +1508,9 @@ def _compute_recurrence_message(self):
             else:
                 recurrence_title = _('A new task will be created on the following dates:')
             task.recurrence_message = '<p><span class="fa fa-check-circle"></span> %s</p><ul>' % recurrence_title
-            task.recurrence_message += ''.join(['<li>%s</li>' % date.strftime(date_format) for date in recurring_dates[:5]])
+            task.recurrence_message += ''.join(['<li>%s %s</li>' % (
+                format_date(date, "EEE", locale=self.env.user.lang), date.strftime(date_format)
+            ) for date in recurring_dates[:5]])
```

Comment:
```suggestion
            date_format = f"%a {self.env['res.lang']._lang_get(self.env.user.lang).date_format}"
            ...
            task.recurrence_message += ''.join(['<li>%s</li>' % date.strftime(date_format))
            for date in recurring_dates[:5]])
```
Just learned it from Vincent. https://www.programiz.com/python-programming/datetime/strftime
`%a` = Abbreviated weekday name (Sun, Mon, ..)

Reply:
If we want the days of the week to be in the language of the user, we would need to change the 'locale' of the program according to the user language, it seems like a bad practice, Vincent said it'll be more convenient to use babel, can you confirm ?

Reply:
No probs :p thanks for the review

Reply:
I am wondering if we cannot use the following function instead:
https://github.com/odoo/odoo/blob/87307a9010c6c872fe6c746d5ece378ad90d038f/odoo/tools/misc.py#L1359-L1392

What do you think? 🙂